src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -16,8 +15,10 @@
     private static final String WRONG_DATA_INSERTION = "Будь ласка, введіть коректні дані";
     private static final String WHITESPACE_SEPARATOR = " ";
     private static final String QUIT_SIGN = "q";
-    private BetDao betDao = new BetDaoImpl();
-    private UserDao<User> userDao = new UserDaoImpl<User>();
+    @Inject
+    private BetDao betDao;
+    @Inject
+    private UserDao<User> userDao;
```

Comment:
mentioned about this in prev PR, make code consistent

Reply:
I don't understand what have i done?

Reply:
done